MesaService/ServiceTasks/HandleUserInactivity.cs

```diff
@@ -17,9 +17,11 @@ public class HandleUserInactivity : IServiceTask
 
         public bool Run()
         {
+            SendDiscordMessage("171373018285735936", "Gamearoo", "Hello Gamearoo This is a message from MesaSuite sent to the bot's api. \n I am just letting you know i ran and if you see this message from the bot it works <:green:1123707359232020520> .");
```

Comment:
So there's a couple issues with this I think.

1) This will send you a message every hour regardless of whether or not the system actually sent messages.
2) Not a huge fan of hard-coding a DiscordID here. Recommend using `ConfigurationManager` as seen lower in this file to grab a DiscordID from the configuration file. You can add the entry in the configuration in MesaService's `App.config` file (see the other config entries in that file).

Reply:
i pushed a change i hope i fixed it to what you where looking for